spec/unit/classes/server/service_spec.rb

```diff
@@ -0,0 +1,32 @@
+
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'postgresql::server::service', type: :class do
+
+  let :pre_condition do
+    'include postgresql::server'
+  end
+
+  let :node_params do
+    {
+      'lsbmajdistrelease' => '18.04',
+    }
+  end
```

Comment:
Where it this used?

Reply:
This is used in `params.pp` Line 168 to determine `$service_name`. The test will fail when the variable is empty.

Reply:
Still, shouldn't that be a fact? I've submitted https://github.com/puppetlabs/puppetlabs-postgresql/pull/1307 for that to be dropped though so I'd prefer to go that route and then rebase this.